src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -110,12 +147,72 @@ module.exports = kind({
 		this.modelIndex = idx;
 		return records;
 	},
+	generateDataGridList: function(itemType) {
+		if (this.$.gridList) {
+			this.$.gridList.destroy();
+		}
+
+		var moreProps = {};
+
+		switch (itemType) {
```

Comment:
It was supposed to have more options than just 2, but I could not make it.
How about leave switch statement for further works?
When app developers have different GUI, I'll add more option for their reference.